### DISCUSSION TITLE
"Releases" not showing up

### DISCUSSION THREAD

**Discussion by stardriftfx** [community/community#6396] [Other Feature Feedback, Questions, & Ideas] [answered]
Not sure if "Releases" works for newer releases, but this error happens on older GitHub projects.

**Comment by MylesBorins** [comment, marked as answer]
@cherrybiird do you have existing releases or only tags? As part of this work we have decoupled releases and tags and no longer show tags that do not have a GitHub release on the releases page.

Could you share what repository you were looking at

**Reply by stardriftfx** [reply]
@MylesBorins I was looking at this repository
https://github.com/LouisBrunner/valgrind-macos/releases

**Reply by MylesBorins** [reply]
@cherrybiird the reason no releases are showing up is because the project has only pushed tags and has not created GitHub Releases. Until they publish releases that section will be empty

**Comment by sometimegithubuser** [comment]
Another example is VIM at [https://github.com/vim/vim-win32-installer/releases] not showing any code.

**Reply by MylesBorins** [reply]
I'm a bit confused... the page you shared has a number of releases listed

**Reply by sometimegithubuser** [reply]
The releases are there, but none have any files listed.
![image](https://user-images.githubusercontent.com/24495871/139129882-c888cd9c-05db-458d-b23a-fa328b58167a.png)
If I click on the Code and the most recent release, it has files listed. 
![image](https://user-images.githubusercontent.com/24495871/139130331-713e17d4-e5c1-40a5-a27d-5e8455d5c873.png)

**Reply by MylesBorins** [reply]
@sometimegithubuser thank you much for the explanation. We have code we are sharing between components to truncate code that is currently removing images from the feed view... we are working on a replacement already and hope to have something out very soon to fix this

**Comment by axelboc** [comment]
I'm not seeing the "Releases" section on [this repo](https://github.com/silx-kit/vscode-h5web) even though I have published 4 releases. Is it because they are v0.0.x releases only? If so, it seems like a weird constraint. :shrug:

**Reply by eafer** [reply]
I was just having this same problem, so for the record in case it helps anyone else: when you are logged in, there's a little wheel right next to "About" and you need to enable "Releases" in there.

**Reply by axelboc** [reply]
Thank you, thank you, thank you!!

**Reply by ThIsLinked** [reply]
@axelboc thanks, men! Its working!

**Reply by codemile** [reply]
Omg. I spent an hour trying to figure out why "Releases" was missing from my 3 year old repo. Thank you. Such a stupid optimization.

**Reply by yannbf** [reply]
Thank you so much for explaining this @eafer. Life saver!! Here's a more visual guide:
![image](https://github.com/user-attachments/assets/f7042d6a-f7d3-41e9-8cca-f2a638757884)

**Comment by danielgomezrico** [comment]
The same thing is happening to me on this repo: https://github.com/la-haus/flutter-segment/pull/20

We had multiple releases and if I go to https://github.com/la-haus/flutter-segment/releases it does show the page but it shows an empty state without showing the old releases :panic:

**Reply by danielgomezrico** [reply]
After I published this comment the releases now appear but... it is empty and there are no tags in the repository neither :O

**Comment by Morsmalleo** [comment]
Same problem here with my own repository, I've published at least 4 releases with source code only, and now the release tab is not showing up, it was showing up before, because I was planning to release a new version that I've got drafted and have been editing after every fix and update Implement, but I can't access the releases section anymore when I have a look this morning, it's gone!

**Reply by Morsmalleo** [reply]
Never mind I fixed it, idiot me accidentally unchecked the releases box, when editing the description of the repository.

**Comment by elespdn** [comment]
Hello, I seem to have a similar problem for this repo: https://github.com/RISE-UNIBAS/TEIworkshop.

Releases exist, but I cannot see them on the repo homepage, even if I check the box in Settings of the About section and receive the message "Your repository details have been saved."

Any hint would be welcome, thanks

**Reply by champsupertramp** [reply]
I have the same issue now

**Reply by foreverforbidden** [reply]
https://github.com/orgs/community/discussions/6396#discussioncomment-9374146 fixes this

**Comment by abhimanyu003** [comment]
I'm having the same issue, there are releases but it's not showing up in UI at the home page. 
https://github.com/abhimanyu003/sttr

**Reply by ThIsLinked** [reply]
At you is all worked.

**Reply by abhimanyu003** [reply]
Hey yes it worked you have to click this cog icon next to about.
Then it will give you options which you want to display on the page.

<img width="849" alt="image" src="https://github.com/community/community/assets/265913/4c8c05b5-43e4-49e7-9f2d-454ef089b1f6">

**Comment by babymonie** [comment]
having the same issue on my repo https://github.com/babymonie/Luar

**Reply by ThIsLinked** [reply]
At you is all worked.

**Comment by lost-scripts** [comment]
My problem was that latest release wasn't showing up in the ***Releases*** section to the right of the repository root. Well, FTR, since everything is currently in beta stage, they were all marked as ***Pre-release*** and it turned out that was the reason... But I kept wanting the latest one to show up there, so it wasn't until I simply <ins>unchecked</ins> the ***"Set as a pre-release"*** checkbox on the latest one that it started showing up 🙂. So there it is just in case anyone finds it helpful!

**Comment by MButcho** [comment]
Nothing above worked for me, but what did was changing visibility back and forth on a repo.

**Comment by sudip-sth** [comment]
My releases used to show up in releases page. I deleted some old tags and releases. Now the releases page shows 
<img width="558" height="310" alt="image" src="https://github.com/user-attachments/assets/ad2764b7-19be-4151-aa03-0e8456ff1a51" />

I have to go to the tags section to view the release notes now. Do we have any solution for this ?

**Comment by siddarthkay** [comment]
I have this same weird issue for https://github.com/status-im/status-app
On homepage it shows there are a bunch of releases : 

<img width="1440" height="859" alt="Screenshot 2025-12-12 at 11 52 05 AM" src="https://github.com/user-attachments/assets/fbc92843-77fb-4013-9291-7b8d889a4d00" />

But when you click the latest release on the homepage it shows there are none ? this is a very obvious bug in Github

<img width="1435" height="635" alt="Screenshot 2025-12-12 at 11 53 21 AM" src="https://github.com/user-attachments/assets/36a2598f-9b7c-4005-83b4-351e406a3915" />

I've even tried disabling and re-enabling releases in this repo but that does not change anything

<img width="945" height="751" alt="Screenshot 2025-12-12 at 11 55 07 AM" src="https://github.com/user-attachments/assets/da370c55-e985-453d-9935-9414102b61d9" />

Any help is greatly appreciated, the team just pushed a new release and we don't want to confuse our users.

**Comment by yurii-github** [comment]
hello

are 5 years not enough to fix this?


Releases like "vX.x" ARE NOT VISIBLE in the list. It only works with format like "vX.x.x" etc.

Release with format "vX.x" is only visible on the front page when it is the latest release.

**Comment by yurii-github** [comment]
Also, when latest  release like "vX.x.x" is deleted, previous latest release like "vX.x" WILL NOT become latest on the front page.

And release counter shows correct number of releases.

p.s.
if you think this will force me to use "vX.x.x" format, then i must disappoint you, i have a bit different idea.
